src/modules/iotjs_module_buffer.c

```diff
@@ -252,6 +252,17 @@ JHANDLER_FUNCTION(Compare) {
 }
 
 
+#define CONVERT_DOUBLE_TO_SIZE(n, d) \
+  size_t n;                          \
```

Comment:
@LaszloLango IMHO, what about moving the declaration out of this macro? Let the user to define their variable before `CONVERT_DOUBLE_TO_SIZE'. 

From
```
CONVERT_TO_DOUBLE_TO_SIZE(dst_start, 0, dst_length);
```

to

```
size_t dst_start;
CONVERT_TO_DOUBLE_TO_SIZE(dst_start, 0, dst_length);
```

Because I was confused where is the declaration of `dst_start`, `src_start`, and so on. I am not used to this style. Moreover, automatically declared variable in `CONVERT_DOUBLE_TO_SIZE` remains after `CONVERT_DOUBLE_TO_SIZE`.

Or alternatively, what about changing the name like `DECLARE_SIZE_T_FROM_DOUBLE` because it does not simply convert, but also declare variable.